src/wasm.h

```diff
@@ -54,12 +54,12 @@ struct FeatureSet {
 
   bool isMVP() const { return features == MVP; }
   bool has(Feature f) { return (features & f) == f; }
-  bool hasAtomics() const { return features & Atomics; }
-  bool hasMutableGlobals() const { return features & MutableGlobals; }
-  bool hasTruncSat() const { return features & TruncSat; }
-  bool hasSIMD() const { return features & SIMD; }
-  bool hasBulkMemory() const { return features & BulkMemory; }
-  bool hasAll() const { return features & All; }
+  bool hasAtomics() const { return static_cast<bool>(features & Atomics); }
```

Comment:
Why is this an improvement?    Do you know which modernize pass did this?

Reply:
I did this because of warnings:

https://ci.appveyor.com/project/WebAssembly/binaryen/builds/22694718

I fixed this error in next commit.